hw2/style-hw2.css

```diff
@@ -0,0 +1,214 @@
+* {
+	padding: 0;
+	margin: 0;
+}
+
+body {
+	background-color: #f9f9f9;
+}
+/*----------header----------*/
+.container {
+	width: 75%;
+	background-color: #fff;
+	margin: 0 auto;
+	box-shadow: 0 0 3px 2px #dbdbdb;
+	padding: 50px 45px 60px 45px;
+}
+
+.main-menu {
+	margin-top: 35px;
+	height: 40px;
+	width: 100%;
+	background: linear-gradient(to top, #444343, #5e5e5e);
+}
+
+.main-menu__item {
+	list-style: none;
+	float: left; 
+	border-right: rgba(255,255,255,0) 1px solid;
+	border-left: rgba(255,255,255,0) 1px solid;
+}
+
+.main-menu__link {
+	color: #ffffff;
+	text-decoration: none;
+	font-family: sans-serif;
```

Comment:
Try to find needed font, include it and use font-stack, Ask @bertaFly of @ValeeraCatcha